tests/prkit-verify.test.sh

```
@@ -50,6 +50,49 @@ from pathlib import Path
 import sys
 p=Path(sys.argv[1]); contract=Path(sys.argv[2]).read_text(); p.write_text(p.read_text()+"\n"+contract)
 PY'
+expect_fail "V13 out-of-scope policy edge fails" 'python3 - "$d/plugins/prkit/policy/solve-run-tree-v1.json" <<'PY'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)